apis/validator/sync_committee_contribution.yaml

```
@@ -4,7 +4,12 @@ get:
     - Validator
   operationId: "produceSyncCommitteeContribution"
   summary: "Produce a sync committee contribution"
-  description: "Requests that the beacon node produce a sync committee contribution."
+  description: |
+    Requests that the beacon node produce a sync committee contribution.
+
+    A 503 error must be returned if the block identified by the response
```

```suggestion
    A 503 error must be returned if the block identified by the request field
```
Also isn't it better to return "400 invalid request"?

---

> Also isn't it better to return "400 invalid request"?

The 503 error is used for "syncing". I figured that having an optimistic block is an artifact of an unsynced node (i.e., the EL is still syncing). I don't feel strongly either way.